src/Shared/LayoutRenderers/AspNetRequestWebSocketNegotiatedProtocolLayoutRenderer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Text;
+using NLog.LayoutRenderers;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Web Socket Negotiated Protocol
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-request-negotiated-protocol}
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-web-socket-negotiated-protocol")]
+    public class AspNetRequestWebSocketNegotiatedProtocolLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
I prefer only LayoutRenderers that are supported by atleast ASP.NET Core

Reply:
Remove until I can find method to make it work for both Framework and Core